homeassistant/components/trafikverket_weatherstation/__init__.py

```diff
@@ -5,23 +5,23 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 
-from .const import DOMAIN, PLATFORMS
+from .const import PLATFORMS
 from .coordinator import TVDataUpdateCoordinator
 
+TVWeatherConfigEntry = ConfigEntry[TVDataUpdateCoordinator]
```

Comment:
As an alternative if you move the type alias into the `coordinator` module, you won't have any circular import issues at all and thus don't need the `if TYPE_CHECKING` block.

Reply:
I personally feel it's the least sensible file to make it so even with the type checking I would prefer to keep it in `__init__.py`